docs/api/resources/system.rst

```diff
@@ -726,6 +726,7 @@ Update resource
                 "stg_language": "en",
                 "stg_directoryservice": "",
                 "stg_guiaddress": "0.0.0.0",
+                "stg_guicertificate": 1,
```

Comment:
Lets add this field down below as well, to the fields description:     :json integer stg_guicertificate ....